spec/models/outing_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'rails_helper'
+
+describe Outing do
+  describe "validations" do
+    context "invalid attributes" do
+      it "is invalid without a title" do
+        outing = Outing.create(description: "thing", image_url: "thingy", base_cost: 4)
+
+        expect(outing).to be_invalid
+      end
+
+      it "is invalid without a description" do
+        outing = Outing.create(title: "thing", image_url: "thingy4", base_cost: 4)
+
+        expect(outing).to be_invalid
+      end
+
+      it "is invalid without an image" do
+        outing = Outing.create(title: "thing", description: "thingy3", base_cost: 4)
+
+        expect(outing).to be_invalid
+      end
+
+      it "is invalid without base cost" do
+        outing = Outing.create(title: "thing", image_url: "thingy1", image_url: "bob")
```

Comment:
@drod1000 Let's fix the duplicated hash key issue before merging into Master. @njgheorghita @drod1000